src/components/TransactionSender/index.ts

```diff
@@ -1,8 +1,9 @@
-import React, { useContext, useRef } from 'react';
+import React, { useRef } from 'react';
 import { Address } from '@elrondnetwork/erdjs';
 import { Signature } from '@elrondnetwork/erdjs/out/signature';
 
-import OverrideDefaultBehaviourContext from 'contexts/OverrideDefaultBehaviourContext';
+import { sendSignedTransactions as defaultSendSignedTxs } from 'apiCalls/transactions';
```

Comment:
Since this is reused, it should probably be moved to types folder

Reply:
it's a function. TBD